dali/pipeline/operators/fused/resize_crop_mirror.h

```diff
@@ -26,6 +26,14 @@
 #include "dali/pipeline/operators/common.h"
 #include "dali/pipeline/operators/crop/crop_attr.h"
 
+#if defined(__AARCH64_QNX__)
+typedef unsigned int uint;
+#endif
+
+#if defined(__AARCH64_QNX__)
```

Comment:
those definitions should be probably moved to `dali/common.h`

Reply:
done